update/update_v3.x.x_to_v4.x.x.sh

```diff
@@ -0,0 +1,234 @@
+#!/usr/bin/env bash
+
+PRG=$(basename -- "${0}")
+USAGE="
+usage: ./${PRG}
+    Required environment variables:
+    IBMCLOUD_API_KEY
```

Comment:
maybe add a blank line here to split up Required environment variables and Dependencies ?